### ISSUE TITLE
`min/max_val` not working in `_scalar_arg_to_dict()`

### ISSUE THREAD

**Issue by maximelucas** [xgi-org/xgi#442]
Running 

```python
import xgi
from xgi.drawing.draw_utils import _scalar_arg_to_dict

H = xgi.random_hypergraph(5, ps=[0.5, 0.05])

_scalar_arg_to_dict(50, H.nodes, min_val=10, max_val=20)
# --> {0: 50, 1: 50, 2: 50, 3: 50, 4: 50}   # all values are 50 > 20, wrong!
```

As a consequence, the default settings in `draw_nodes()`
```python
      "min_node_size": 10.0,
      "max_node_size": 30.0,
      "min_node_lw": 1.0,
      "max_node_lw": 5.0,
```

and all other default min/max settings going through `_scalar_arg_to_dict` are ignored.

**Comment by maximelucas** [comment]
https://github.com/xgi-org/xgi/pull/441 solved this for drawing nodes. Still needs fixing for drawing hyperedges.

**Comment by maximelucas** [comment]
#456 fixed this for hyperdges. The functions `_color_arg_to_dict()` and `_scalar_arg_to_dict()` are not used in the new versions, but are still used in `draw_dihypergraph()`, `draw_multilayer()`.

**Comment by maximelucas** [comment]
Fixed by #476 for `draw_dihypergraph` by removing dependency. `draw_multilayer` remains.
